tests/test_util.py

```diff
@@ -13,3 +14,14 @@ def test_get_latest_version_timeout(mock_pypi_request_timeout):
     start = time()
     assert get_latest_version() is None
     assert (time() - start) < 0.5
+
+
+@pytest.mark.usefixtures("enable_docker_pull")
+def test_image_is_latest(docker_client):
+    """Test that the latest version is identified correctly."""
+    # download the alpine image for testing
+    image_name = "alpine:latest"
+    docker_client.images.pull(image_name)
+
+    # check that the image is identified as latest
```

Comment:
It would be good to also test the converse, i.e. pull some outdated version of the same image and check that it is identified as not latest. That should be possible right? (should be a separate test to keep it clean)

Reply:
I think it can, by pulling the outdated one based on the digest.